src/lightning/fabric/loggers/csv_logs.py

```diff
@@ -62,7 +63,8 @@ def __init__(
         flush_logs_every_n_steps: int = 100,
     ):
         super().__init__()
-        self._root_dir = os.fspath(root_dir)
+        self.fs, _ = fsspec.core.url_to_fs(str(root_dir))
```

Comment:
Let's do this instead to match the TensorBoard implementation

https://github.com/Lightning-AI/lightning/blob/afc63e9f665add07625f3845fdf45716b72e294e/src/lightning/fabric/loggers/tensorboard.py#L105

```suggestion
        self._fs = get_filesystem(root_dir)
```